src/mjolnir.py

```diff
@@ -13,20 +13,16 @@
 import actions
 import synapse
 from charm_state import CharmState
-from constants import (
-    MJOLNIR_MANAGEMENT_ROOM,
-    MJOLNIR_MEMBERSHIP_ROOM,
-    MJOLNIR_SERVICE_NAME,
-    PEER_RELATION_NAME,
-    SECRET_ID,
-    SECRET_KEY,
-    SYNAPSE_CONTAINER_NAME,
-)
 from user import User
 
 logger = logging.getLogger(__name__)
 
 USERNAME = "mjolnir"
+# Disabling it since these are not hardcoded password
+SECRET_ID = "secret-id"  # nosec
```

Comment:
Can you order the constants alphabetically?

Reply:
thanks, they are sorted across all files that defines constant (I finally found an editor extension to do that quickly)